news/compute_selection_fixes.rst

```diff
@@ -0,0 +1,27 @@
+**Added:**
+
+* OpenMMEngineSettings now has a `gpu_device_index` attribute
+  allowing users to pass through a list of ints to select the
+  GPU devices to run their simulations on.
+
+**Changed:**
+
+* `openfe.protocols.openmm_rfe._rfe_utils.compute` has been moved
+  to `openfe.protocols.openmm_utils.omm_compute`.
```

Comment:
Do we want a private namespace or is this now in our public API?
```suggestion
  to `openfe.protocols.openmm_utils._omm_compute`.
```

Reply:
I'd say public developer API is fine, private was because we were directly vendoring from perses.